webpack/src/Extends/Host/PuppetTab/SubTabs/YamlTab/YamlTab.scss

```diff
@@ -0,0 +1,3 @@
+.yaml-tab {
+  padding: 16px 24px;
+}
```

Comment:
for a one-line CSS rule I would remove this file and just do it inline in that div

Reply:
Removed the css file and added inline css.